TicTacToeAppTests/GamePromptSpec.swift

```diff
@@ -0,0 +1,50 @@
+
+import Quick
+import Nimble
+import TicTacToeApp
+
+class GamePromptSpec: QuickSpec {
+  override func spec() {
+
+    func makeMovesInSequence(game: Game, moveSequence: [Int]) {
+      for spaceId in moveSequence {
+        game.makeMove(spaceId)
+      }
+    }
+
+    describe("GamePrompt") {
```

Comment:
We should plan on discussing possible improvements in how to organize your tests in describes.